### PR TITLE
2024 Batch

### PR DESCRIPTION
**Pull request by jacobbullock**
Adding Deep Dish Swift and NY Swift

### CODE REVIEW

**Review by davedelong — APPROVED** [review]

File: _data/conferences.yml [hunk 1]
```diff
@@ -23,6 +23,26 @@
...
+
+- name: Deep Dish Swift
+  link: https://deepdishswift.com
+  start: 2024-05-05
+  end: 2024-05-07
+  location: 🇺🇸 Chicago, USA
+  cocoa-only: true
+  cfp:
+    link: https://www.papercall.io/deep-dish-swift-2024
+    deadline: 2023-01-31
```
Comment (pardel) [inline _data/conferences.yml:44]: @jacobbullock the deadline might be Jan 2024 rather than 2023 😄

Comment (davedelong) [reply _data/conferences.yml:44]: Yeah I corrected that already. My mistake for missing this before merging.

Comment (pardel) [reply _data/conferences.yml:44]: Sorry, I've only noticed it now when trying to update it :D

**Comment by Lascorbe** [comment]
thank you all @jacobbullock, @pardel and @davedelong! ❤️

**Pull request merged**